js/ui/grid_core/ui.grid_core.data_controller.js

```diff
@@ -713,7 +713,10 @@ module.exports = {
                     }
 
                     if(item1.rowType === 'group' || item1.rowType === 'groupFooter') {
-                        if(item1.isExpanded !== item2.isExpanded || JSON.stringify(item1.summaryCells) !== JSON.stringify(item2.summaryCells)) {
+                        const expandedMatch = item1.isExpanded === item2.isExpanded;
+                        const summaryCellsMatch = JSON.stringify(item1.summaryCells) === JSON.stringify(item2.summaryCells);
+                        const continuationMatch = typeUtils.isDefined(item1.data) && typeUtils.isDefined(item2.data) ? item1.data.isContinuation === item2.data.isContinuation : true;
```

Comment:
Add isContinuationOnNextPage match
Also it is possible to use "?." instead of isDefined.

Reply:
OK